packages/aws-cdk-lib/aws-efs/lib/efs-file-system.ts

```diff
@@ -586,7 +586,7 @@ export class FileSystem extends FileSystemBase {
       performanceMode: props.performanceMode,
       throughputMode: props.throughputMode,
       provisionedThroughputInMibps: props.provisionedThroughputPerSecond?.toMebibytes(),
-      backupPolicy: props.enableAutomaticBackups ? { status: 'ENABLED' } : undefined,
+      backupPolicy: { status: props.enableAutomaticBackups ? 'ENABLED' : 'DISABLED' },
```

Comment:
Can we also have unit test and integ test for this change?

Reply:
Yes, it will cause a breaking change. The documentation says it's false by default, but it actually is always treated as true (see #29881).

Reply:
Yeah, Since this involves a breaking change, we need to handle via the feature flag. [CDK Feature Flags](https://docs.aws.amazon.com/cdk/v2/guide/featureflags.html) are a mechanism that allows the CDK to evolve and change the behavior of certain classes and methods, without causing disruption to existing deployed infrastructure.

Below are the steps on adding a feature flag:
1. Define the feature flag in [features.ts](https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/cx-api/lib/features.ts#L1040-L1049) 
2. Update the new feature flag in [FEATURE_FLAGS.md](https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/cx-api/FEATURE_FLAGS.md)
3. Implement the feature flag. For reference [here](https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-ec2/lib/launch-template.ts#L648-L656)